integration_test/README.md

```diff
@@ -64,15 +64,16 @@ To complete installation, the following command are required
 
-| Script                                   | Description                                                                                                                                                                               |
-|------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `same_device_integration_test.py`        | Simulates a same-device authentication flow with an OpenID4VP relying party.                                                                                                              |
-| `cross_device_integration_test.py`       | Simulates a cross-device authentication flow, e.g., mobile to desktop.                                                                                                                    |
-| `same_device_integration_test_duckle.py` | Simulates a same-device authentication flow with an OpenID4VP relying party and Duckle credentials.                                                                                       |
-| `user_denies_end_to_end_test.py`         | Simulates the end-to-end flow where the user explicitly denies sharing credentials in the wallet, testing the OpenID4VP Authorization Error Response flow (same-device and cross-device). |
+| Script                                   | Configurations                                                                                                                                                                                                                                                                                | Description                                                                                                                                                                               |
+|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 🛑 `same_device_integration_test.py`     | To run this integration test, you need to modify the [pyeudiw_backend.yaml](./conf/pyeudiw_backend.yaml) configuration file by removing the following entries: `config.duckle.dcql_query`. Additionally, you must remove the DuckleHandler in the `credential_presentation_handlers` section. | Simulates a same-device authentication flow with an OpenID4VP relying party.                                                                                                              |
```

Comment:
why?

we need to test DCQL not removing it

PE is now deprecated, we cannot think to continue using it.

DCQL MUST not be removed since it MUST be used within the integration tests